atom/browser/ui/message_box_mac.mm

```diff
@@ -157,10 +157,13 @@ int ShowMessageBox(NativeWindow* parent_window,
           callEndModal:true];
 
   NSWindow* window = parent_window->GetNativeWindow().GetNativeNSWindow();
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
   [alert beginSheetModalForWindow:window
                     modalDelegate:delegate
                    didEndSelector:@selector(alertDidEnd:returnCode:contextInfo:)
                       contextInfo:nil];
```

Comment:
can we instead switch to using [beginSheetModalForWindow:completionHandler:](https://developer.apple.com/documentation/appkit/nsalert/1524296-beginsheetmodalforwindow?language=objc)? here and below

Reply:
Yup that seems like the alternative, can we change this as a follow up ?